src/lib/types.ts

```diff
@@ -16,6 +16,11 @@ export interface Options {
   savePEM: boolean;
 }
 
+export interface MutableAuthorizeRedirectUri {
```

Comment:
Why not just use [`URL`](https://nodejs.org/dist/latest-v14.x/docs/api/url.html#url_class_url)? Seems more versatile.

Reply:
@markwallsgrove kind bump?

Reply:
@poveden :+1: